README.md

```diff
@@ -87,7 +87,7 @@ import EmberValidations from 'ember-validations';
 
 export default Ember.Component.extend(EmberValidations, {
   validations: {
-    'user.firstName': {
+    'model.user.firstName': {
```

Comment:
This one doesn't need to be `model.user.firstName` as it looks like the intent here is to operate from the `user` attribute so `user.firstName` works